src/components/articles/cbd-article.vue

```diff
@@ -35,7 +35,7 @@
 
 import axios from 'axios';
 import ArticlesApi from '../../services/api/articles';
-import {lstring } from '../../services/filters/lstring'
+import { lstring } from '../../services/util'
```

Comment:
I think the former version was better. 
```javascript
import lstring from '../../services/filters/lstring
```
I have a preference for importing only what we need and not rely on a general import/export.
I think the goal is to expose everyhting to a general use librabry But internally we should import only the the neede element

I will discuss this with @blaisefonseca